src/pages/es/core-concepts/astro-components.md

```diff
@@ -180,33 +192,71 @@ const items = ["Perro", "Gato", "Mono"];
 </ul>
 ```
 
-Los Fragments también pueden ser útiles para evitar envolver componentes al usar las directivas [`set:*`](/es/reference/directives-reference/#sethtml), como en el siguiente ejemplo:
+Los fragmentos también pueden ser útiles para evitar envolver componentes al usar las directivas [`set:*`](/es/reference/directives-reference/#sethtml), como en el siguiente ejemplo:
 
-```astro
+```astro title="src/components/SetHtml.astro" "Fragment"
 ---
 const htmlString = '<p>Contenido HTML sin procesar</p>';
 ---
 <Fragment set:html={htmlString} />
 ```
 
-### Props de componentes
+### Diferencias entre Astro y JSX
+
+La sintaxis del componente Astro es un superconjunto de HTML. Fue diseñado para que resulte familiar para alguien con experiencia en HTML o JSX, pero hay un par de diferencias clave entre los archivos `.astro` y JSX.
+
+#### Atributos
+
+En Astro, utiliza el formato estándar `kebab-case` para todos los atributos HTML en lugar del `camelCase` que se usa en JSX. Esto incluso funciona para `class`, que no es compatible con React.
+
+```jsx del={1} ins={2} title="example.astro"
+<div className="box" dataValue="3" />
+<div class="box" data-value="3" />
+```
+
+#### Modifying `<head>`
+
+En JSX, existen bibliotecas especiales para ayudarlo a administrar la etiquetas `<head>` de la página. Esto no es necesario en Astro. Escriba `<head>` y su contenido en un layout de nivel superior.
```

Comment:
Although `biblioteca` is the translation for `library`, it is more common to use `librería` (a false friend, which means `book shop`). I have no proof for this besides my personal impression.

```suggestion
En JSX, existen librerias especiales para ayudarlo a administrar la etiquetas `<head>` de la página. Esto no es necesario en Astro. Escriba `<head>` y su contenido en un layout de nivel superior.
```